<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWorkReport</class>
 <widget class="QDialog" name="EditWorkReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Новый отчёт о работе</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/icons/icons/report_edit.png</normalon>
   </iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Работа:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="eOrderID">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Дата:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="eDate">
       <property name="inputMethodHints">
        <set>Qt::ImhDate</set>
       </property>
       <property name="displayFormat">
        <string>dd.MM.yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Мастер:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="eMaster">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="bCancel">
       <property name="text">
        <string>Отменить</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/icons/icons/door_out.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bSave">
       <property name="text">
        <string>Сохранить и закрыть</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/icons/icons/page_save.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="eWork">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Стоимость:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="eWPrice">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="bAddWork">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/icons/icons/add.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Кол-во:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="eQuant">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="maxLength">
        <number>11</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Номер заказа:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>Запчасти:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Запчасть:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="eSpare">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Цена:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" colspan="2">
       <widget class="QLineEdit" name="eSPrice">
        <property name="maximumSize">
         <size>
          <width>101</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QToolButton" name="bAddSpare">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/icons/icons/add.png</normalon>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QPushButton" name="bUse">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/icons/icons/brick_add.png</normalon>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="4" colspan="2">
       <widget class="QPushButton" name="bRemove">
        <property name="text">
         <string>Удалить</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/icons/icons/brick_delete.png</normalon>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="6">
       <widget class="QTableWidget" name="tview">
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <column>
         <property name="text">
          <string>Наименование</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Цена</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Кол-во</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
